framework-plugins/lisk-framework-dashboard-plugin/package.json

```diff
@@ -34,11 +34,12 @@
 		"prebuild": "rm -r dist-node/* || mkdir dist-node || true",
 		"build": "npm run build:node && npm run build:web",
 		"build:node": "tsc",
-		"build:web": "node scripts/build.js --jsx react",
-		"start:web": "NODE_ENV=development node scripts/start.js",
+		"build:web": "parcel build --dist-dir build public/index.html",
```

Comment:
why usage of `build` command twice 🤔

Reply:
second one is specifying the `dist-dir` first one is part of the command `parcel build`